engine/crates/integration-tests/src/engine/builder.rs

```diff
@@ -96,6 +103,10 @@ impl EngineBuilder {
             schema_builder = enable_local_dynamo(schema_builder).await;
         }
 
+        if self.local_postgres {
+            schema_builder = schema_builder.data(runtime_local::LocalPgTransportFactory::runtime_factory());
+        }
```

Comment:
Do you think it's worth just having local_postgres on all the time?

Dynamo has a specific `with_local_dynamo` function because we need to setup dynamo tables for the tests where we're using dynamo and I didn't want to do that on every single test.  But looks like adding this to the context doesn't have much of a cost so we could just do it all the time?

Dunno, up to you

Reply:
aye sounds good